Mouse.h

```diff
@@ -25,7 +25,7 @@ class Mouse {
     Maze maze;
     Hardware bot;
 
-    Coordinate position;
+    NodeCoordinate position;
```

Comment:
If we want to allow diagonals then the mouse would need to be in between cells.
![image](https://cloud.githubusercontent.com/assets/4276762/23621780/e64bea7a-0250-11e7-8bed-e5e181fade35.png)